src/api/commands/user-managment.ts

```diff
@@ -31,24 +31,22 @@ import { UserManagmentConstants } from "../constants/user-management-constants";
  * @param  {boolean} failOnStatusCode- Whether to fail on response codes other than 2xx and 3xx
  * */
 Cypress.Commands.add("createUserViaAPI", (host: string, username: string, password: string, reqBody: Cypress.ObjectLike,
-     grantType: string, authType: "Basic" | "Bearer", failOnStatusCode = true) => {
+    grantType: string, authType: "Basic" | "Bearer", failOnStatusCode = true) => {
 
-    cy.getAuthentication(host, username,
-        password, grantType, authType).then(response => {
+    cy.getAuthentication(host, username, password, grantType, authType).then(response => {
 
-            const token: string = response.body.access_token;
+        const token: string = response.body.access_token;
 
-            return cy.request({
-                "method": RequestMethodTypes.POST,
-                "url": host + UserManagmentConstants.SCIM2_ENDPOINT + UserManagmentConstants.SCIM2_USER_ENDPOINT,
-                "failOnStatusCode": failOnStatusCode,
-                "auth": {
-                    token
-                },
-                "headers": {
-                    "Content-Type": RequestContentTypes.URLENCODED
-                },
-                "body": reqBody
-            });
+        return cy.request({
+            "method": RequestType.POST,
+            "url": host + UserManagmentConstants.SCIM2_ENDPOINT + UserManagmentConstants.SCIM2_USER_ENDPOINT,
+            "failOnStatusCode": failOnStatusCode,
+            "headers": {
+                "Content-Type": RequestContentTypes.JSON,
+                "accept": RequestContentTypes.JSON,
+                "Authorization": token
```

Comment:
Are we getting the token or the full header value (eg: "Bearer <token>") here?

If we are getting the header value, it'd be better to rename the variable.